pylint_import_requirements/__init__.py

```diff
@@ -183,6 +171,13 @@ def check_module_without_origin(self, node, spec, names: Optional[List[str]]):
         self.add_message("missing-requirement", node=node, args=(spec.name, alternative_dist_msg))
         return
 
+    def _is_stdlib_or_first_party_module(self, package):
+        """Check if the given path is from a built-in module or not"""
+
+        # Approach taken from https://github.com/PyCQA/pylint/blob/master/pylint/checkers/imports.py
+        import_category = self.isort_obj.place_module(package)
+        return import_category in {"FUTURE", "STDLIB", "FIRSTPARTY"}
```

Comment:
just for reference (no need to change):
```python
>>> import isort
>>> isort_obj = isort.SortImports(file_contents="")
>>> isort_obj.place_module('sys')
'STDLIB'
>>> isort_obj.place_module('json')
'STDLIB'
>>> isort_obj.place_module('numpy')
'THIRDPARTY'
>>> isort_obj.place_module('not_numpy')
'FIRSTPARTY'
```

module not found => 'FIRSTPARTY'

so we rely on pylint's import-error check here